core/src/main/java/zipkin2/reporter/internal/AsyncReporter.java

```diff
@@ -273,7 +278,7 @@ void flush(BufferNextMessage<S> bundler) {
       });
 
       try {
-        sender.sendSpans(nextMessage).execute();
```

Comment:
this was the only place where the sender was used for its real purpose in this repo. So, the net change here is just more efficient and straightforward of the same